contracts/lib/Iden3Helpers.sol

```diff
@@ -0,0 +1,323 @@
+pragma solidity ^0.5.0;
+
+import './Memory.sol';
+
+/**
+ * @dev Define interface Mimc7 contract
+ */
+contract MimcUnit {
+  function MiMCpe7(uint256,uint256) public pure returns(uint256) {}
+}
+
+/**
+ * @dev Iden3 helper functions
+ */
+contract Iden3Helpers {
```

Comment:
The functions in this file seem to be outated.  We no longer use mimc7.  We don't use claims that authorize ethereum keys.  Claims are 8 entries.

Reply:
Yes, Iden3Helpers are from the old spec, is not used currently, probably will be deleted at the end of the current phase (not yet, as now is in current active development of the branch)